src/server/modes/charts/plugins/datalens/utils/axis-helpers.ts

```diff
@@ -60,11 +60,28 @@ export const applyPlaceholderSettingsToAxis = (
             placeholder.settings.autoscale === false ||
             (placeholder.settings.scale === 'auto' && placeholder.settings.scaleValue === '0-max')
         ) {
-            axis.min = 0;
+            if (placeholder.settings.type === 'logarithmic') {
+                // Fallback to null for incorrect case
+                axis.min = null;
+            } else {
+                axis.min = 0;
+            }
         } else if (placeholder.settings.scale === 'manual') {
             axis.endOnTick = false;
             axis.min = Number((placeholder.settings.scaleValue as [string, string])[0]);
             axis.max = Number((placeholder.settings.scaleValue as [string, string])[1]);
+
+            if (placeholder.settings.type === 'logarithmic') {
+                if (axis.min <= 0) {
+                    // Fallback to null for incorrect case
+                    axis.min = null;
+                }
+
+                if (axis.max <= 0) {
+                    // Fallback to null for incorrect case
+                    axis.max = null;
+                }
+            }
```

Comment:
Perhaps it is worth adding a denial of this to the condition? To simply not set any values and leave the default.

The above is similar